src/server/lobby-server.ts

```diff
@@ -25,6 +25,9 @@ export class LobbyServer {
     console.log("Players in lobby: ", this.players.length);
 
     this.resetTimer();
+    for (var playerSocket of this.players) {
+      playerSocket.emit('new player', this.players.length);
```

Comment:
Any interop stuff like `'new player'` should be a constant in `server-interface.ts` or `client-interface.ts`.   `server-interface.ts` contains all the interfaces and strings for data/message types that it sends to the client.
`client-interface` contains the interfaces and strings for data/message types that it sends to the server